framework/validators/EmailValidator.php

```diff
@@ -24,13 +24,13 @@ class EmailValidator extends Validator
      * @var string the regular expression used to validate the attribute value.
      * @see http://www.regular-expressions.info/email.html
      */
-    public $pattern = '/^[a-zA-Z0-9!#$%&\'*+\\/=?^_`{|}~-]+(?:\.[a-zA-Z0-9!#$%&\'*+\\/=?^_`{|}~-]+)*@(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?\.)+[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?$/';
+    public $pattern = '/(*UTF8)^[\pL0-9!#$%&\'*+\\/=?^_`{|}~-]+(?:\.[\pL0-9!#$%&\'*+\\/=?^_`{|}~-]+)*@(?:[\pL0-9](?:[\pL0-9-]*[\pL0-9])?\.)+[\pL0-9](?:[\pL0-9-]*[\pL0-9])?$/';
```

Comment:
@samdark yeah, you're right, we can use /u instead, (*UTF8) is just advice from PCRE http://www.pcre.org/pcre.txt
will fix it